pkg/controllers/cspi-controller/controller.go

```diff
@@ -92,12 +93,16 @@ func NewCStorPoolInstanceController(
 	eventBroadcaster.StartRecordingToSink(&typedcorev1.EventSinkImpl{Interface: kubeclientset.CoreV1().Events("")})
 	recorder := eventBroadcaster.NewRecorder(scheme.Scheme, corev1.EventSource{Component: poolControllerName})
 
+	// Initilizes the new instance of Zcommand to execute zpool/zfs commands
+	zcmdExecutor := zcmd.NewZcmd()
```

Comment:
n_p: Typo in `Initilizes`
Also, instead of `Initialises` I would use `instantiate`.

Reply:
Updated